lua/core/autopairs.lua

```diff
@@ -1,54 +1,63 @@
--- if not package.loaded['nvim-autopairs'] then
---   return
--- end
-local Log = require "core.log"
-local status_ok, _ = pcall(require, "nvim-autopairs")
-if not status_ok then
-  Log:get_default().error "Failed to load autopairs"
-  return
+local M = {}
+
+function M.config()
+  lvim.builtin.autopairs = {
+    active = true,
+    ---@usage  map <CR> on insert mode
+    map_cr = true,
+    ---@usage auto insert after select function or method item,
+    map_complete = vim.bo.filetype ~= "tex",
```

Comment:
Maybe add a note above this as it only works if the first file you open is a tex file, not when switching from Markdown to tex for example.

Reply:
Yeah I'm not sure how any of this works. Feel free to write the note you'd like to see here, and I'll add it verbatim.

Reply:
Smth like
`-- NOTE: This should be wrapped into a function so that it is re-evaluated when opening new files`